.github/workflows/CI.yml

```
@@ -63,6 +63,52 @@ jobs:
       - uses: codecov/codecov-action@v3
         with:
           files: lcov.info
+  python:
+    name: ${{ matrix.test_group }} / Julia ${{ matrix.version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
```

```suggestion
          - '1.11'
```
Isn't that what we want?

---

just setting '1' is ok as it means "get me the latest 1.x version"